### PR TITLE
handle Multiple

### PR DESCRIPTION
**Pull request by ravalravi103**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -6,11 +6,43 @@ import './App.css';
...
+            return ({
+              ...item,completed :!item.completed
+            })
+          }
+          else {
+            return ({
+              ...item, completed : item.completed
+            })
+          }
+          return item
```
Comment (shahsank3t) [inline src/App.js]: You are already returning in `if` and `else` then no need to return the item here.
Also, you can make it better by removing  else statement and doing something like
```
if(condtion){
   return ...
}
return ...
```

File: src/Todo.js [hunk 2]
```diff
@@ -19,13 +19,29 @@ function Todo({text,todo,todos,setTodos}) {
          }))
      }
 
+
+     // check Handler
+     const checkHandler = (e) => {  
+         console.log(todo.id)
+           setTodos(todos.map(item =>{
```
Comment (shahsank3t) [inline src/Todo.js]: Do we need to loop over the complete todos here?  Can't we use the index and update the check?

**Review by shahsank3t — CHANGES_REQUESTED** [review]

File: src/Form.js [hunk 3]
```diff
@@ -8,7 +8,7 @@ function Form({ inputText,setInputText,todos,setTodos}) {
 
     const submitTodoHnadler = (e) => {
          setTodos([
-             ...todos,{text:inputText,completed: false,id: Math.random()*1000}
+             ...todos,{text:inputText,completed: false, check : false,id: Math.random()*1000}
```
Comment (chirag-madlani) [inline src/Form.js:11]: Why we need random IDs?

File: src/Todo.js (hunk 2 above)
Comment (chirag-madlani) [reply src/Todo.js]: We have IDs too, Can we use that?

**Pull request merged**